examples/organization_lock/main.tf

```diff
@@ -0,0 +1,24 @@
+terraform {
+  required_version = ">= 0.13.0"
+  required_providers {
+    oasis = {
+      source = "arangodb.com/managed/oasis"
+      version = "1.5.1"
+    }
+  }
+}
+
+provider "oasis" {
+  api_key_id     = "" // API Key ID generated in Oasis platform
+  api_key_secret = "" // API Key Secret generated in Oasis platform
+}
+
+// Terraform created organization
+resource "oasis_organization" "oasis_test_organization" {
+  name        = "Terraform Oasis Organization"
+  description = "A test Oasis organization from Terraform Provider"
```

Comment:
Why not simple set locked attribute on organization ?

Reply:
We had discussed this with Ewout yesterday and decided to do the locking on a separate resource locking Terraform resource, but today we agreed to do it on Organization level. I also think it's more intuitive to have in on Organization level. The previous decision was made for explicitness (comparing it to Oasisctl). Everything is updated now to be on Organization (/resource) level.